ChangeLog

```diff
@@ -6,10 +6,41 @@ Pydot versions since 1.4.2 adhere to [PEP 440-style semantic versioning]
 (https://www.python.org/dev/peps/pep-0440/#semantic-versioning).
 
 
-3.0.3 (unreleased)
+4.0.0 (unreleased)
 ------------------
 
-- Nothing changed yet.
+Added:
+- Added initial type annotations. (#430)
+  There are still a lot of `Any`s, but Pydot now officially supports type annotations.
+
+Changed:
+- Simplify `Common.get_top_graph_type` with a default. (#437)
+- Define parser elements in a separate `GraphParser` class. (#464)
+  It's a user-transparent change, but now it's easier to debug, test, and modify the parser.
+  See the related Pull Request for more information.
+
+Fixed:
+- Avoid subprocess window creation within `call_graphviz` on Windows. (#431)
+- Removed a blank line inserted after subgraphs in the `to_string()` output. (#434)
+- Fixed "all numeric" quoting behavior. (#439)
```

Comment:
This now appears in both the 4.0.0 changelog and the 3.0.3 changelog. That may even be correct, I honestly can't remember: Did I fix it twice? :rofl: